modules/admin_manual/pages/maintenance/upgrading/upgrade.adoc

```diff
@@ -30,6 +30,8 @@ There are two ways to upgrade your ownCloud server:
 
 1. *(Recommended)* Perform a xref:maintenance/manual_upgrade.adoc[manual upgrade], using the
 https://owncloud.com/download-server/[latest ownCloud release].
-2. *(Discouraged)* Use xref:maintenance/package_upgrade.adoc[your distribution’s package manager], in conjunction with our official ownCloud repositories. *Note:* This approach should not be used unattended nor in clustered setups. We discourage upgrades with Linux Package Manager because you might encounter unwanted side effects.
+2. *(Recommended for simple installations)* Click the [Open Updater] button in the Admin General page. This button is available, when the online updater has detected a new version available. 
```

Comment:
I would maybe not use the word "Recommended" here but rather "Alternatively to the recommended approach for single-server setups" or something similar.

"There are two ways to upgrade your ownCloud server" needs also to be edited if this option is added.

Reply:
Good suggestions. Added.